lib/log_ratelimit.c

```diff
@@ -132,3 +137,34 @@ rte_log_ratelimit(uint32_t level, uint32_t logtype, const char *format, ...)
 
 	return 0;
 }
+
+void
+set_log_level_per_block(const char *block_name, uint32_t log_level)
+{
+	for (int i = 0; i < RTE_MAX_LCORE; i++) {
+		if(strcmp(log_ratelimit_states[i].block_name,
+					block_name) == 0) {
+			rte_atomic32_set(&log_ratelimit_states[i].log_level,
```

Comment:
Do we have an extra TAB before `block_name) == 0) {`?

Reply:
Yes, it was automatically added. Removed the extra TAB.